Procedures/Affiliated-Package-Review-Process.md

```diff
@@ -279,6 +279,13 @@ included some comments when the score is not green.
 
 *Include any other comments here*
 
+*If accepted and there are no governance documents or only partial governance documents:*
+Please remember that if your package doesn't explicitly have a set
+of governance documents, that's fine!  The PlanetaryPy Project's
+governance models (consensus seeking, code of conduct, etc.)
+automatically apply to your project.  If you want something different,
+be sure to include it in your pacakge.
+
```

Comment:
package (you really don't like that word, do you? ;) )

Reply:
I'm just not used to running the spell-checker on these files.  Fixed.